doc/source/contributing.rst

```diff
@@ -595,15 +595,15 @@ Running the test suite
 The tests can then be run directly inside your Git clone (without having to
 install *pandas*) by typing::
 
-    nosetests pandas
+    pytest pandas
 
 The tests suite is exhaustive and takes around 20 minutes to run.  Often it is
 worth running only a subset of tests first around your changes before running the
 entire suite.  This is done using one of the following constructs::
 
-    nosetests pandas/tests/[test-module].py
-    nosetests pandas/tests/[test-module].py:[TestClass]
-    nosetests pandas/tests/[test-module].py:[TestClass].[test_method]
+    pytest pandas/tests/[test-module].py
+    pytest pandas/tests/[test-module].py::[TestClass]
```

Comment:
I would instead recommend using ``-k``, in fact this is one of the main reasons I really like ``pytest``, e.g.

``pytest pandas/[tests-module].py -k cool_test`` IMHO is much easiest then finding class names and the exact names of the methods.